blueprints/validations.py

```diff
@@ -0,0 +1,49 @@
+"""Module for validation actions inside of Blueprints."""
+
+
+class NonPositiveValueError(Exception):
+    """Raised when a value is less than or equal to zero."""
```

Comment:
In my opinion it is quite confusing to use "Litotes" in error messages, but I can't think of a better one so just leave it.

Reply:
Maybe then LessOrEqualToZeroError???